cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/runtime/batch/MapReduceRuntimeService.java

```diff
@@ -348,6 +348,10 @@ protected void startUp() throws Exception {
         Transactions.invalidateQuietly(txClient, tx);
         throw t;
       }
+    } catch (LinkageError e) {
```

Comment:
Probably its not required in the `destroy` method because contract of the `destroy` is that it does not throw any exception, we catch `Throwable` and just log error.